code/modules/mob/living/simple_animal/hostile/mining_mobs/gutlunch.dm

```diff
@@ -145,9 +145,8 @@
 		L = new /mob/living/simple_animal/hostile/asteroid/gutlunch/gubbuck(loc)
 	else
 		L = new /mob/living/simple_animal/hostile/asteroid/gutlunch/guthen(loc)
-	L.key = key //I don't think this needs logging
-	L.mind = mind
-	L.faction = faction
```

Comment:
factions are mobbound and are not transferred with the mind unfortunately unless you meant to do this.

Reply:
I didn't mean to delete that...